### ISSUE TITLE
Hybrid - no more battery autonomy but level is not 0

### ISSUE THREAD

**Issue by coelhoricardo** [flobz/psa_car_controller#426]
**Describe the bug**
Instead having a level at 0, we have level at 100.
After restarting HA addon, now it's at level 40

**To Reproduce**
1. i just use get_vehicleinfo for Home Assistant
2. when the bug append ? using the car (level changed from 1 to 100) or idle (level changed from 100 to 40)
3. Does it persist after restarting the program? yes (i restarted HA addon)
4. Does it persist after recreate config by going to http://server_address:port/config? not tested

**Config file**
(Give the anonymize content of the config file)

**Environment (please complete the following information):**
 - OS: Home Assistant VM
 - C5 Aircross Hybrid
 - v3.0.5 (HA addon)
 - (Log from http://server_address:port/log and browser console log)

**Additional context**
get_vehicleinfo (level 40 and autonomy 0, actually it should be level 0)
```
  "energy": [
    {
      "updated_at": "2022-11-10 10:34:56+00:00",
      "autonomy": 0,
      "battery": null,
      "charging": {
        "charging_mode": "No",
        "charging_rate": 0,
        "next_delayed_time": "PT23H30M",
        "plugged": false,
        "remaining_time": null,
        "status": "Disconnected"
      },
      "consumption": null,
      "level": 40,
      "residual": null,
      "type": "Electric"
    },
    {
      "updated_at": "2022-11-10 10:34:56+00:00",
      "autonomy": 370,
      "battery": null,
      "charging": null,
      "consumption": null,
      "level": 56,
      "residual": null,
      "type": "Fuel"
    }
  ],
```

**Comment by coelhoricardo** [comment]
Other weird things happened, apart from psa car controller, after I created this ticket.
In fact, even on MyCitroen app, it said 100% with a autonomy of 0km.
My wife plugged the car in yesterday with a delayed charge for 23:30.
But this morning the car was not charged.
From MyCitroen, I forced the charge. It stopped a few minutes later. The car still indicated that the charge was delayed until 23:30.
After unplugging and replugging the home charger, the charge didn't start either.
When I opened the car, unplugged and plugged in the charger, it started to charge. After several minutes, the charge level finally went from 100% to 4%.

All this is happening, coincidentally, since the car reached 0% for the first time since the installation of "psa car controller".
